metadata/0.json

```diff
@@ -0,0 +1 @@
+{"locationId":"loc-id"}
```

Comment:
[ultra-nit] add space at EOF

Reply:
idk why this is even changing will look into it